src/server/game/Spells/SpellInfo.h

```diff
@@ -432,6 +432,9 @@ struct SpellPowerCost
 {
     Powers Power;
     int32 Amount;
+
```

Comment:
Is there a reason for this change?

Reply:
I had a piece of code locally removed that used it and forgot to return it.
Returned now.